src/main/java/com/github/onsdigital/dp/image/api/client/RetryStrategy.java

```diff
@@ -0,0 +1,35 @@
+package com.github.onsdigital.dp.image.api.client;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.ServiceUnavailableRetryStrategy;
+import org.apache.http.protocol.HttpContext;
+
+/**
+ * Custom implementation of ServiceUnavailableRetryStrategy to retry any HTTP 5xx responses.
+ */
+public class RetryStrategy implements ServiceUnavailableRetryStrategy {
+
+    private final int maxRetries;
+    private final long retryIntervalMs;
+
+    public RetryStrategy(int maxRetries, long retryIntervalMs) {
+        this.maxRetries = maxRetries;
+        this.retryIntervalMs = retryIntervalMs;
+    }
+
+    public RetryStrategy() {
+        this(3, 20);
+    }
+
+    @Override
+    public boolean retryRequest(HttpResponse response, int executionCount, HttpContext context) {
+        return executionCount <= maxRetries &&
+                response.getStatusLine().getStatusCode() >= HttpStatus.SC_INTERNAL_SERVER_ERROR;
```

Comment:
This is a very good question. This class is pretty much a copy of [the equivalent in the dataset api client](https://github.com/ONSdigital/dp-dataset-api-java-client/blob/master/src/main/java/dp/api/dataset/RetryStrategy.java) and I haven't given it much thought. I'll consider this with a tech lead.

Reply:
Having discussed this with the tech leads we're going to keep the functionality for now consistent with the dataset client and will revisit the subject of retries more holistically as a separate piece of work.

Reply:
👍 I did see the conversation. I asked the question because I wanted to clarify that was the intended behaviour. If that is what has been decided then I will resolve this comment.